stubs/backend-proxy.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const express = require("express");
+const http = require("http");
+const httpProxy = require('http-proxy');
+
+const assemblyProxy = httpProxy.createProxyServer({
+  host: 'http://localhost:4631'
+});
+
+const documentsProxy = httpProxy.createProxyServer({
+  host: 'http://localhost:4603'
+});
+
+const idamToken = require('../bin/IdamHelper').getIdamToken();
```

Comment:
This method returns a promise so you need an await here and "top level" await is not supported.

Take the contents of the file, except the imports and wrap them in an async function and then call that.

For example:

https://github.com/planarnetwork/eurostar2gtfs/blob/master/src/cli.ts